velox/functions/prestosql/aggregates/AverageAggregate.cpp

```diff
@@ -78,10 +78,11 @@ class AverageAggregate : public exec::Aggregate {
     double* rawSums = sumVector->mutableRawValues();
     for (auto i = 0; i < numGroups; ++i) {
       char* group = groups[i];
+      clearNull(rawNulls, i);
       if (isNull(group)) {
-        rowVector->setNull(i, true);
+        sumVector->setNull(i, true);
+        rawCounts[i] = 0;
```

Comment:
This is to make partial avg has aligned result with duckdb. Can be removed if you think it's not needed.